lms/djangoapps/mobile_api/models.py

```diff
@@ -25,3 +25,67 @@ def get_video_profiles(cls):
         Get the list of profiles in priority order when requesting from VAL
         """
         return [profile.strip() for profile in cls.current().video_profiles.split(",") if profile]
+
+
+class MobileAppVersionConfig(ConfigurationModel):  # pylint: disable=model-missing-unicode
+    """
+    Configuration for mobile app versions available.
+    """
+    latest_version_android = TextField(
+        blank=False,
+        help_text="Latest available version for android app in X.X.X format"
+    )
+    latest_version_ios = TextField(
+        blank=False,
+        help_text="Latest available version for IOS app in X.X.X format"
+    )
+    min_supported_version_android = TextField(
+        blank=False,
+        help_text="Min supported version for android app in X.X.X format"
+    )
+    min_supported_version_ios = TextField(
+        blank=False,
+        help_text="Min supported version for IOS app in X.X.X format"
+    )
+    next_supported_version_android = TextField(
```

Comment:
`next_supported` is kind of misleading, because other versions are supported too. I would recommend we make it `next_min_supported`. That will eliminate the need to explain the variable.